data/json/items/comestibles/egg.json

```
@@ -742,21 +742,22 @@
   {
     "type": "COMESTIBLE",
     "id": "rehydrated_eggs",
-    "copy-from": "egg_bird",
-    "name": { "str": "rehydrated egg" },
+    "name": { "str_sp": "liquid eggs" },
```

```suggestion
    "name": { "str_sp": "rehydrated eggs" },
```
In America liquid eggs mean the below product which has removed the yolks.  https://www.target.com/p/cage-free-liquid-egg-substitute-32oz-good-38-gather-8482/-/A-79519411?ref=tgt_adv_xsp&AFID=google&fndsrc=tgtao&DFA=71700000108264730&CPNG=PLA_Dairy%2BFrozen_Local%7CDairy%2BFrozen_Ecomm_Food_Bev&adgroup=SC_Dairy%2BFrozen&LID=700000001170770pgs&LNM=PRODUCT_GROUP&network=g&device=c&location=1025952&targetid=pla-511168984938&gclid=CjwKCAjw9pGjBhB-EiwAa5jl3BrIMTPjR99SwyNro12Bwy94BM96s6FtMRhv1zyCqjiEd_y8Wu6zWRoCO2MQAvD_BwE&gclsrc=aw.ds